create-oci.sh

```diff
@@ -74,15 +99,31 @@ for artifact_pair in "${artifact_pairs[@]}"; do
 
     # log "creating tar archive %s with files from %s" "${archive}" "${path}"
 
-    if [ ! -r "${path}" ]; then
-        # non-existent paths result in empty archives
-        tar "${tar_opts[@]}" "${archive}" --files-from /dev/null
-    elif [ -d "${path}" ]; then
-        # archive the whole directory
-        tar "${tar_opts[@]}" "${archive}" --directory="${path}" .
+    if [ -z "${no_tar}" ]; then
+        if [ ! -r "${path}" ]; then
+            # non-existent paths result in empty archives
+            tar "${tar_opts[@]}" "${archive}" --files-from /dev/null
+        elif [ -d "${path}" ]; then
+            # archive the whole directory
+            tar "${tar_opts[@]}" "${archive}" --directory="${path}" .
+        else
+            # archive a single file
+            tar "${tar_opts[@]}" "${archive}" --directory="${path%/*}" "${path##*/}"
+        fi
     else
-        # archive a single file
-        tar "${tar_opts[@]}" "${archive}" --directory="${path%/*}" "${path##*/}"
+        artifact_name="$(basename ${path})"
+        archive="${archive_dir}/${artifact_name}"
+        if [ ! -r "${path}" ]; then
+            # non-existent paths result in empty files
+            echo -n "{}" > ${archive}
```

Comment:
```suggestion
            # non-existent paths result in empty files
            touch "${archive}"
```

Reply:
I had `touch` originally, but then I changed it as the OCI spec seems to want at least a valid empty json object by default. We are not enforcing the default file name, though, so the type will not necessarily be json. I can change it back to `touch`.